### PR TITLE
Fix lint issues

### PR DESCRIPTION
**Pull request by klzns**
#### What problem is this solving?

Cleaning the house.

#### How to test it?

https://lint--storecomponents.myvtex.com/jobs-notebook/p

### CODE REVIEW

File: react/components/SearchBar/SearchBar.tsx [hunk 1]
```diff
@@ -252,13 +252,10 @@ function SearchBar({
             openMenu,
           }) => (
             <div
-              className={classNames(
-                'relative-m w-100',
-                withModifiers('searchBarInnerContainer', [
```
Comment (klzns) [inline react/components/SearchBar/SearchBar.tsx:256]: That's really weird, `classNames` was transforming the array in a single string `opened filled` making it an invalid modifier and not applying them.

**Review by vitorflg — APPROVED** [review]
👏

**Review by igorbrasileiro — APPROVED** [review]

File: react/components/SearchBar/SearchBar.tsx (hunk 1 above)
Comment (igorbrasileiro) [reply react/components/SearchBar/SearchBar.tsx:256]: :thonking:

**Review by icazevedo — APPROVED** [review]
:statisfaction:

**Review by victorhmp — APPROVED** [review]

**Pull request merged**